src/SDKs/Network/Network.Tests/Tests/ExpressRouteTests.cs

```diff
@@ -130,5 +134,44 @@ public void ExpressRouteMicrosoftPeeringApiTest()
                 resourcesClient.ResourceGroups.Delete(resourceGroupName);
             }
         }
+
+        [Fact]
+        public void ExpressRouteMicrosoftPeeringApiWithIpv6Test()
+        {
+            var handler1 = new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK };
+            var handler2 = new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK };
+
+            using (MockContext context = MockContext.Start(this.GetType().FullName))
+            {
+                var resourcesClient = ResourcesManagementTestUtilities.GetResourceManagementClientWithHandler(context, handler1);
```

Comment:
@vinayada1 please use latest resource manager, you are still using the old resource manager.